sauron/sauron.py

```diff
@@ -90,27 +87,30 @@ def getchaininfo(plugin, **kwargs):
         raise SauronError("Unsupported network")
     plugin.sauron_network = chains[genesis_req.text]
 
-    # We wouldn't be able to hit it if its bitcoind wasn't synced, so
-    # ibd = false and headercount = blockcount
```

Comment:
Why remove this comment?

Reply:
That was just a mistake. I´ll add it again.